saleor/graphql/notifications/inputs.py

```diff
@@ -0,0 +1,11 @@
+import graphene
+
+
+class ExternalNotificationTriggerInput(graphene.InputObjectType):
+    ids = graphene.List(graphene.ID, required=True, description="List of customer ids.")
```

Comment:
It's not a list of customer ids. It's a list of node IDs that will be serialized and included in the notification payload. It should be mentioned in the description.